Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
```

Comment:
Any reason to use an ubuntu image rather than `python:3-alpine` so you don't have to install python and the image can be smaller?

https://hub.docker.com/_/python

Reply:
The function pthread_attr_setaffinity_np is not on alpine linux and needed for pytorch. I am not sure if there is a slimmer distro than ubuntu. Open to try something if you have any suggestions.

Reply:
I'd prefer we use ubuntu and maybe even package the model weights in the dockerfile (you're going to have to download it anyway, and we pin the revision in docquery).

Reply:
On my local when using it to develop I mount my machines huggingface cache. But I think as a general easy to use docker container that would simplify things and be respectful of huggingfaces resources.

Reply:
Do we want to build from source than? Or should I just pip install docquery?

Reply:
I think it depends on how we want to publish it. We should probably get a better release cadence going and pin the Dockerfile to whatever the latest release is. I think the way to do that would be to somehow provide the version as a parameter to the Dockerfile and populate the parameter by looking at https://github.com/impira/docquery/blob/main/src/docquery/version.py.

Reply:
`docker run IMAGE_NAME docquery scan "What is the invoice number?" https://templates.invoicehome.com/invoice-template-us-neat-750px.png`

Seems like a really nice way to use the tool.